readme.md

```diff
@@ -432,11 +432,7 @@ is.any(is.boolean, 'unicorns', [], new Map());
 //=> false
 ```
 
-See below for use with multiple predicates.
-
-##### .any(predicates[], ...values)
-
-Returns `true` if **any** of the input `values` returns true in **any** of the `predicates` provided in an array:
+Using an array of `predicates`, returns `true` if **any** of the input `values` returns true in **any** of the `predicates` provided in an array:
```

Comment:
```suggestion
Using an array of `predicates`, returns `true` if **any** of the input `values` returns true for **any** of the `predicates` provided in an array:
```

Reply:
Pushed a new commit to include the word change with previous changes: https://github.com/sindresorhus/is/pull/104/commits/b662c7cddcf5691984165ad4a3711fbecf03baa6